infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/VerticalGeometryListing.kt

```diff
@@ -0,0 +1,158 @@
+package fi.fta.geoviite.infra.geometry
+
+import fi.fta.geoviite.infra.common.AlignmentName
+import fi.fta.geoviite.infra.common.DomainId
+import fi.fta.geoviite.infra.common.StringId
+import fi.fta.geoviite.infra.common.TrackMeter
+import fi.fta.geoviite.infra.geocoding.GeocodingContext
+import fi.fta.geoviite.infra.geography.Transformation
+import fi.fta.geoviite.infra.math.*
+import fi.fta.geoviite.infra.util.FileName
+import kotlin.math.tan
+
+data class CurvedSectionEndpoint(
+    val address: TrackMeter?,
+    val height: Double,
+    val angle: Double?,
+    val station: Double,
+)
+
+data class IntersectionPoint(
+    val address: TrackMeter?,
+    val height: Double?,
+    val station: Double?,
```

Comment:
Voisiko nämä kaksi jälkimmäistä olla non-null? Tätä dataa käyttävän kaverin puolesta kyselen.

Reply:
Hyvä huomio, ainakin suunnitelmapuolella nää voi olla non-nulleja. Taidan lähteä puksuttamaan eteenpäin tuon paikannuspohjapuolen haun kanssa ja korjata nämä sitten siellä.